dartagnan/src/main/java/com/dat3m/dartagnan/Dartagnan.java

```diff
@@ -112,19 +113,20 @@ public static void main(String[] args) throws Exception {
         File fileProgram = new File(Arrays.stream(args).filter(a -> supportedFormats.stream().anyMatch(a::endsWith))
                 .findFirst()
                 .orElseThrow(() -> new IllegalArgumentException("Input program not given or format not recognized")));
-        logger.info("Program path: " + fileProgram);
+        logger.info("Program path: {}", fileProgram);
 
         File fileModel = new File(Arrays.stream(args).filter(a -> a.endsWith(".cat")).findFirst()
                 .orElseThrow(() -> new IllegalArgumentException("CAT model not given or format not recognized")));
-        logger.info("CAT file path: " + fileModel);
+        logger.info("CAT file path: {}", fileModel);
 
-        Wmm mcm = new ParserCat().parse(fileModel);
+
+        Wmm mcm = new ParserCat(Path.of(o.getCatIncludePath())).parse(fileModel);
```

Comment:
I don't think this works if you add an include path different than where the main cat file is, e.g., `java -jar dartagnan/target/dartagnan.jar cat/tso.cat --cat.include=/home/ponce/git/Dat3M/ litmus/AARCH64/extra2/IRIW+poaps+LL.litmus`

Reply:
Yes, because all includes will be resolved w.r.t. the provided inclusion path so `basic.cat` won't be found.
What solution do you suggest? A include resolution strategy that checks multiple locations?
I think creating a robust resolution strategy is not easy.

Reply:
Are you saying that here the problem is that it cannot find `basic.cat` in `--cat.include=/home/ponce/git/Dat3M/`? How is this different from not finding e.g., `filters.cat` (from `tso.cat`) when not passing `--cat.include` and just logging a warning?

I am fine with "all includes to be assumed to be in the same path", what I don't want is the assumption that the model file I pass as `cat/X.cat` should also be in the include path and that the line I commented above looks like.

Am I missing something?

Reply:
This cmd does not work
`java -jar dartagnan/target/dartagnan.jar cat/tso.cat --cat.include=/home/ponce/git/Dat3M litmus/AARCH64/extra2/IRIW+poaps+LL.litmus`
this one does
`java -jar dartagnan/target/dartagnan.jar cat/tso.cat --cat.include=/home/ponce/git/Dat3M/cat litmus/AARCH64/extra2/IRIW+poaps+LL.litmus`

> if I provide an invalid path it fails to include
fails means you see a warning or you get an exception?

Reply:
I get an exception due to usage of undefined relations (e.g. `fr`). Which is to be expected because `basic.cat` fails to include.